src/main/telemetry/srxl.c

```diff
@@ -161,8 +166,20 @@ typedef struct
 } STRU_TELE_FP_MAH;
 */
 
-void srxlFrameFlightPackCurrent(sbuf_t *dst)
+#define FP_MAH_KEEPALIVE_TIME_OUT 2000 // mS
+
+bool srxlFrameFlightPackCurrent(sbuf_t *dst, timeUs_t currentTimeUs)
 {
+  uint16_t amps = getAmperage() / 10;
+  uint16_t caps = getMAhDrawn();
+  static uint16_t sentAmps;
+  static uint16_t sentCaps;
+  static timeUs_t lastTimeSentFPmAh = 0;
+
+  timeUs_t keepAlive = (currentTimeUs - lastTimeSentFPmAh)/1000;
+
+  if ( (amps != sentAmps) || (caps != sentCaps) ||
+       keepAlive > FP_MAH_KEEPALIVE_TIME_OUT ) {
     sbufWriteU8(dst, SRXL_FRAMETYPE_TELE_FP_MAH);
     sbufWriteU8(dst, SRXL_FRAMETYPE_SID);
     sbufWriteU16(dst, getAmperage() / 10);
```

Comment:
amps / caps

Reply:
mah now as of above.

Reply:
No. You're have value in variable, use it, don't fetch it again

Reply:
Ahh, yes of course.